protostar/compiler/project_cairo_path_builder.py

```diff
@@ -1,7 +1,14 @@
 from pathlib import Path
 
 
-class ProjectCairoPathBuilder:
+class LinkedLibrariesBuilder:
+    def build_project_cairo_path_list(
+        self, relative_cairo_path_list: list[Path]
+    ) -> list[Path]:
+        return relative_cairo_path_list
+
+
+class ProjectCairoPathBuilder(LinkedLibrariesBuilder):
```

Comment:
Do we need this inheritance here? What does it give us?

Reply:
We do not need it here, but it is hard to remove for now and makes cope of the task much bigger
First i want to remove cairo0 runner
Then we can work on https://github.com/software-mansion/protostar/issues/1559 which includes the change you suggested